src/config/appConfig.js

```diff
@@ -1,44 +1,35 @@
 // Variable to configure if PWA Service Worker is registered in index.js.
 const pwaActive = false;
-let tilesUrl = '//tiles.dev.trafimage.geops.ch';
 const geoadminWmtsUrl = '//maps{1-3}.trafimage.ch';
 const geoserverUrl = '//maps.trafimage.ch/geoserver/trafimage/ows';
 let tileserverUrlMapproxy = '//wkp.dev.trafimage.geops.ch/raster';
-let vectorTilesUrl = 'https://maps.geops.io';
-const vectorTilesKey =
-  '5cc87b12d7c5370001c1d6557f01e26728174c1fa19d33afe303b910';
+let vectorTilesKey = '5cc87b12d7c5370001c1d6557f01e26728174c1fa19d33afe303b910';
+let vectorTilesUrl = '//maps.geops.io';
```

Comment:
it didn't work on localhost without https , @gevos do you see the tiles in local ?

Reply:
@oterral Yes, I can see the tiles in local.